Makefile

```diff
@@ -287,5 +287,12 @@ endif
 	shellcheck test/extras/*.sh
 	run-parts --exit-on-error --regex '.sh' test/lint
 
+.PHONY: staticcheck
+staticcheck:
+ifeq ($(shell command -v staticcheck),)
+	(cd / ; go install -v -x honnef.co/go/tools/cmd/staticcheck@latest)
+endif
+	staticcheck $$(git ls-files *.go | sed 's|^|./|; s|/[^/]\+\.go$$||' | sort -u)
```

Comment:
Please can you add a comment here explaining what the sed does. 
This both helps future readers quickly understand what is going on and helps reviewers to valid the code itself.